src/ape/managers/project.py

```diff
@@ -285,11 +285,15 @@ def load_contracts(
         # If a file is deleted from ``sources`` but is in
         # ``cached_sources``, remove its corresponding ``contract_types`` by
         # using ``ContractType.source_id`` and ``ContractType.sourcePath``
-        deleted_sources = cached_sources.keys() - set(map(str, sources))
+        deleted_source_ids = cached_sources.keys() - set(
```

Comment:
The changes in this file should have been part of the PR but I didn't realize it had broken. I added a test that fails now without this. Basically, the contracts were not getting returning properly because `ape` thought were always deleted.